src/components/reusables/Form.tsx

```diff
@@ -33,8 +40,10 @@ export default class Form extends React.Component<FormProps, void> {
           />
         }
         { this.props.content }
-        <SaveButton
+        <br />
```

Comment:
No way to not use a `<br />`? I find them harder to maintain than updating style rules in css.

Reply:
I think the `<br />` is more likely to work no matter what kind of elements someone has passed to the Form component.